iam-policy-tls-cert-backup-s3.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_iam_policy" "tls_cert_backup_s3" {
+  provider = aws.clientaccount
+  for_each = aws_route53_zone.clusters
+  name     = "tls-cert-backup-s3-${aws_route53_zone.clusters[each.key].name}"
+  policy   = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:PutObject",
+        "s3:GetObject*",
```

Comment:
Does the backup need these  GET permissions?

Reply:
not neccessarily. will remove it.